stable/aws-pod-identity-webhook/README.md

```diff
@@ -0,0 +1,53 @@
+# AWS Pod Identity Webhook
+
+This chart will install the [Amazon EKS Pod Identity Webhook](https://github.com/aws/amazon-eks-pod-identity-webhook). This tool allows you to specify IAM Roles for Kubernetes Service Accounts. This allows a pod to assume a IAM role.
+
+Further details can be found here: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html
+
+## Prerequisites
+
+- Kubernetes 1.13+
+
+## Installing the Chart
+
+You first need to retrieve `ca.crt` from your cluster as this is used as a value for the chart:
+
+```shell
+secret_name=$(kubectl get sa default -o jsonpath='{.secrets[0].name}')
+export CA_BUNDLE=$(kubectl get secret/$secret_name -o jsonpath='{.data.ca\.crt}' | tr -d '\n')
+```
+
+Then install the chart:
+
+```shell
+$ helm install --name my-release stable/aws-pod-identity-webhook --set caBundle="${CA_BUNDLE}"
+```
+
+After installation you need to approve the certificate. Follow the chart notes after installation for this step.
```

Comment:
Cool. I added a note.

Would be great to have something better here in the chart itself. Perhaps a cronjob. Or something in the webhook itself?

Reply:
I'd lean toward a cronjob that can only approve rather than something in the webhook. Its pretty scary to grant something the ability to create _and_ approve certs

Reply:
> I'd lean toward a cronjob that can only approve

Makes sense. I'll leave this for a later PR though.